src/main/java/com/jalasoft/jfc/model/Param.java

```diff
@@ -83,11 +83,11 @@ public void setInputPathFile(String inputPathFile) {
   }
 
   /**
-   * Sets input file name without extension.
+   * setFolderName sets folder name.
```

Comment:
It is not necessary write method's name "setFolderName sets folder name" -> **Sets folder name**